### DISCUSSION TITLE
Experience of using J4 admin template on wide screen in comparison to J3

### DISCUSSION THREAD

**Discussion by Sulpher** [joomla/joomla-cms#32549] [General]
Hello everybody,
I recently installed J4 beta7 and here I'd like to share my experience of using J4 admin template and problems I met.

I understand that the template is designed to meet the expectations of mobile/gadget users. But they are consumers of the ready site when all is done and you can manage articles.
Let's have a look at the process from another side: of a webmaster.

I am using the wide monitor and love to work in big resolution.

QUICK ACCESS
Well, in fact, there is no too big need in top menu, but its big advantage was a matter of habit and grouped items when you can quickly find what you need. For instance, I want to install extension.
In J3 I go to Extensions > Install extension.
<img width="938" alt="Снимок экрана 2021-02-28 в 14 38 42" src="https://user-images.githubusercontent.com/3432048/109416991-adbefc80-79d2-11eb-955d-1f29cb05d3fa.png">

You can say that dropdown menu is obsolete, but using it I see only related items and quickly go to the menu item I need.

Let's see J4 when we go to the System menu.
<img width="2146" alt="Снимок экрана 2021-02-28 в 14 36 45" src="https://user-images.githubusercontent.com/3432048/109417019-d941e700-79d2-11eb-80bd-0ce9cd82d423.png">

The menu items are structured, but the list is a HUGE! Each time I need to quickly read numerous names to find what I need until it won't became a habit. But the brain analyses the information and you spend more seconds.
So, there are too many menu items and when I making a new site, I need quickly go to Plugins.

The best solution I could find now is to add a custom menu to the sidebar. Here is it:
<img width="760" alt="Снимок экрана 2021-02-28 в 14 42 20" src="https://user-images.githubusercontent.com/3432048/109417099-3b9ae780-79d3-11eb-9750-e15abeef0c27.png">
Using such approach I do not need to waste time on examining the huge list of items on System page.

CREATE MENU ITEMS
here is how it looks like:

<img width="2144" alt="Снимок экрана 2021-02-28 в 14 45 10" src="https://user-images.githubusercontent.com/3432048/109417240-d3003a80-79d3-11eb-813a-a3790802e0d1.png">

It's a loooong distance from Selectbox and to the elements:
I click on Select button, then  modal window appear, I have to move button to the left to select component, then to the right to click on Select again and select category, then move to the left to the toolbar to save the changes.

<img width="2144" alt="Снимок экрана 2021-02-28 в 14 45 10" src="https://user-images.githubusercontent.com/3432048/109417406-a1d43a00-79d4-11eb-8a89-00565f93a3fe.png">

<img width="2117" alt="Снимок экрана 2021-02-28 в 14 49 36" src="https://user-images.githubusercontent.com/3432048/109417412-a698ee00-79d4-11eb-9b1e-aab863aa47c1.png">
<img width="2133" alt="Снимок экрана 2021-02-28 в 14 49 45" src="https://user-images.githubusercontent.com/3432048/109417413-a7318480-79d4-11eb-92a1-1b51614153cc.png">
<img width="2146" alt="Снимок экрана 2021-02-28 в 14 49 57" src="https://user-images.githubusercontent.com/3432048/109417414-a862b180-79d4-11eb-96b7-599bf0205919.png">

Too many moves need to be performed on big resolution screen.
And see how it's simple on J3:
<img width="2148" alt="Снимок экрана 2021-02-28 в 14 44 44" src="https://user-images.githubusercontent.com/3432048/109417448-cdefbb00-79d4-11eb-895b-95e52d79fb5d.png">

<img width="2140" alt="Снимок экрана 2021-02-28 в 14 53 53" src="https://user-images.githubusercontent.com/3432048/109417459-d1834200-79d4-11eb-815b-f94dce7364cc.png">

All is near one area (to the left), the selectbox is not too large. It's comfortable to make a few moves by mouse in a short distance. It keeps time which absolutely matters when you create sites and there is a fight for each second!

Let's make something like that:
![aq3-Untitled-1](https://user-images.githubusercontent.com/3432048/109417557-35a60600-79d5-11eb-898c-4fb51d826f24.png)
Yep, there is an empty space in the middle of the screen, but this is a backend template and we should think about a comfortable work on any resolution for each person!

TOOLBAR
I need to make an extra click on opening a dropdown list with toolbar items.
<img width="2148" alt="Снимок экрана 2021-02-28 в 14 59 52" src="https://user-images.githubusercontent.com/3432048/109417695-c5e44b00-79d5-11eb-959c-6f52e6d6569b.png">

<img width="2145" alt="Снимок экрана 2021-02-28 в 15 02 12" src="https://user-images.githubusercontent.com/3432048/109417749-f6c48000-79d5-11eb-8d45-8a79179bf81f.png">

it's ok for gadgets or laptops, but it needs to perform extra action on the widescreen. 
It's possible to check screen resolution and display a list of items without dropbox on wide screens.
<img width="2148" alt="Снимок экрана 2021-02-28 в 15 02 59" src="https://user-images.githubusercontent.com/3432048/109417820-3ee3a280-79d6-11eb-9abf-00d039d82198.png">
How it could be user-friendly just to see a horizontal list of action buttons! 
And it's shouldn't be too hard to implement it, I hope. let it be dropdown list as now on other devices.

FILTERS
<img width="1858" alt="Снимок экрана 2021-02-28 в 15 06 00" src="https://user-images.githubusercontent.com/3432048/109417924-b9142700-79d6-11eb-8df9-92375429bab0.png">
Compare:
<img width="2146" alt="Снимок экрана 2021-02-28 в 15 08 07" src="https://user-images.githubusercontent.com/3432048/109417937-c9c49d00-79d6-11eb-9156-33a7d82cd365.png">

EDIT ARTICLE
<img width="579" alt="Снимок экрана 2021-02-28 в 15 13 46" src="https://user-images.githubusercontent.com/3432048/109418093-95051580-79d7-11eb-8413-8a93934d432e.png">
Please make black arrow on green background as a white arrow.

As we can see - filters are under the items list. It needs to do the minimal move for the mouse to access what we need.
The same is expected in the new admin template.
Just align filters to the left side under the list of items.

That's all for now. I hope these thoughts can help to make admin template better.
Thanks for your attention.

**Comment by PhilETaylor** [comment]
My discussion on large screen use was completely ignored since I raised it https://github.com/joomla/joomla-cms/discussions/31915

**Reply by Sulpher** [reply]
A good addition to this discussion or vice versa.
Well... The devs awaited users' feedbacks. Here we are. I hope our suggestions will be considered and fixes be implemented in the final release.

**Comment by trogladyte** [comment]
I'm with sulpher on this topic. For a desktop user, this new layout is grossly inefficient. Way too much extra clicking to do the most basic tasks. Why not add a toggle to allow the "classic" top bar menu like in Joomla 1-3? Why do developers think they need to fix what isn't broken? I understand from reading here that the changes were made to help those using mobile devices to edit. Fair enough, assuming that's a big enough percentage to offset annoying desktop users. Was a survey ever done to determine the %age of mobile v desktop users?

I, too, made a custom menu to accomplish the extremely common tasks that the new layout buries. However, I don't see a way to export/import this, so it's 15-20 minutes wasted for every site to set it up.

And, I suppose again to appease the mobile users, every element is at least twice (or 3) times as high as in Joomla 3. Take System | Global Configuration | Site. 2 rolls of the mouse scroll wheel to go through all items v... oh my goodness, ALL of it viewable without scrolling.

From my experience, I have to give the new layout 5 thumbs down. It's very inefficient and the location of items makes no sense in so many occasions. Again, please bring back the "classic" Joomla menu we all know, just as it is. Make it user selectable, but please don't fic what ain't broke!

**Comment by rinka88** [comment]
Dear developers, please take a look at this section. And review what is written

**Comment by leoalv** [comment]
As we can see in the image below there is a problem with the order of the fields, for a simple user this is going to go crazy. The problem worsens if the fields are organized in groups, because grid breaks the order in the user view who does not know if the next field is the one on the bottom or the one on the right side. This is more complicated when `SHOWON` has been used, because when activating the option the field is shown in another column and when there is one more field many times the parent field moves it to another column.

In the images we can see that the lines that were organized in Joomla 3, in J4 no longer even make sense

![J4_bad-UX](https://user-images.githubusercontent.com/5317182/113443882-6c967f80-93b8-11eb-8855-d3946491d5bb.png)
![J4_bad-UX2](https://user-images.githubusercontent.com/5317182/113443897-71f3ca00-93b8-11eb-990d-b677c4f50add.png)

**Reply by brianteeman** [reply]
Please test with the nightly version or direct from github. this has already been resolved

**Reply by leoalv** [reply]
Thanks @brianteeman
